sunpy/net/tests/test_helioviewer.py

```diff
@@ -60,3 +60,29 @@ def test_download_jp2(self, client):
                                        measurement='continuum')
         map_ = sunpy.map.Map(filepath)
         assert isinstance(map_, sunpy.map.GenericMap)
+
+    @skip_glymur
+    def test_download_jp2_permission_error(self, client):
+        """Tests getJP2Image API method"""
+        with pytest.raises(PermissionError):
+            client.download_jp2(
+                '2020/01/01',
+                observatory='SOHO',
+                instrument='MDI',
+                detector='MDI',
+                measurement='continuum',
+                directory='/')
```

Comment:
Would this be a problem on windows machines?

Reply:
yeah probably. you could use `os.path.sep` but I don't think `\` exists on windows either...